src/hidimstat/stat_tools.py

```diff
@@ -460,3 +464,83 @@ def step_down_max_t(stat, permutation_stats):
     two_sided_pval_corr = np.copy(two_sided_pval_corr[stat_ranked])
 
     return two_sided_pval_corr
+
+
+def aggregate_medians(list_one_sided_pval):
```

Comment:
This is just a special case of quantile aggregation. Why create a special function for that ?

Reply:
I was just moved the function for multi_sample_split to stat_tools.

Reply:
By looking into details, I forgot to go through this function.
I discovered that the paper cited in the docstring provides a different equation that the one implemented.
In [Meinshausen et al. 2009](https://arxiv.org/pdf/0811.2177), for aggregation_median, they provide the equation: 
![image](https://github.com/user-attachments/assets/bdee68a2-67cf-4af4-bd60-c43cff67d0de)
where $q_\gamma$ (·) is the (empirical) $\gamma$-quantile function and $\gamma=0.5$ for the median
The code is based on the following equation:
![image](https://github.com/user-attachments/assets/269a6bb0-645d-4eed-b8bb-abc98e23de52)

This difference is also in the function aggregation_quantiles.
Do you think that I should correct it or not?

Reply:
I will generalise the function. 
Do we keep an option for the two-sided tests or do we stay to the initial formula of Meinshausen et al. ?

The same question will also be posed for the function aggregate_quantiles which are the same function but with an automatic selection of the quantile.

Reply:
The question of two-sided tests should not be handled here, but in the procedure that leads to p-values.

Reply:
So we stick to the canonical Meinshausen et al. procedure.

Reply:
For the aggregation function, I think that it's required to know if the p-value is one side or two sides.

If you stick with Meinshausen et al. procedure, I suppose that it's only one-sided tests. In this case, I don't have actually the competence to modify the code with an aggregation function one-sided test for the two-sided tests.

Reply:
No, a p-value is just the probability that the outcome  could be more extreme than the one you got. This is the same definition whether the test is one-sided or two-sided: this pertains to the test specification, and the p-value is the quantification of the test outcome. 
What induces you to think otherwise ?

Reply:
The way to use the term one-side or two-side for me, induce that in on case, only one extreme value of one side of the Gaussian distribution is consider and for two side, this mean that the extreme value of the two side are considerate. Taking this into consideration, the value of the p-value can change drastically if one or two sides are considered.

Reply:
But what do you do with this useless `aggregate_medians` ? Do you want to remove it in a future PR ?

Reply:
This method is used in the `ensemble_clustered_inference_pvalue` for the aggregation of the results across bootstraps.
The default is method is `aggregate_quantiles` which has the same issue about the two-sided setting.

From your discussion, `aggregate_quantiles` and `aggregate_medians` need to be modified to avoid the two-sided setting. 
I can stick to the equation of [Meinshausen et al. 2009](https://arxiv.org/pdf/0811.2177) for fixing it but this will modify the behaviour of the functions.

Reply:
I know but in fact, there is a difference between `aggregate_medians` and `aggregate_quantiles`. 
The function `aggregate_medians` requires to choose the quantile at the difference of the `aggregate_quantiles` which automatically select the quantile depending on the error control desired.
The renaming is to clarify this difference.

Reply:
> For the two-sided thing, I propose to handle it in another PR. What we have currently is not good I'm afraid.

OK, from my understanding, in this PR, we will modify it to only handle one side as the paper of [Meinshausen et al. 2009](https://arxiv.org/pdf/0811.2177) proposed.

Reply:
Thx for the clarification. Regarding `aggregate_quantiles`, I think we've done something wrong. This should correspond to the basic, non-adaptive version of quantile aggregation, of which aggregate_medians is a special case. we also need an adaptive version, e.g. `adaptive_ aggregate_quantiles` for the adaptive version. Can be done in another PR.

Reply:
Bzw, naming should be improved:
`aggregate_quantiles` -> `quantile_aggregation`
and
`adaptive_quantile_aggregation`

Reply:
I realise that the function quantile_aggregation is already implemented in oils. 
I decided to merge the two functions.

Reply:
@bthirion

Reply:
@bthirion

Reply:
Fine. My suggestion is to merge that one and then proceed to further improvements.